src/components/Editor/EditorMenu.js

```diff
@@ -96,7 +102,7 @@ function getMenuItems(
     id: "node-menu-blackbox",
     label: toggleBlackBoxLabel,
     accesskey: blackboxKey,
-    disabled: false,
+    disabled: isOriginal,
     click: () => toggleBlackBox(selectedSource.toJS())
```

Comment:
true, we don't want to blackbox original sources. This is because we don't know how to blackbox an original source that is part of a larger bundled (generated) source